src/JJMasterData.Core/DataDictionary/Repository/DataDictionaryDAO.cs

```diff
@@ -10,11 +10,11 @@
 
 namespace JJMasterData.Core.DataDictionary.Repository;
 
-public class DataDictionaryDao : IDataDictionaryRepository
+public class DataDictionaryDAO : IDataDictionaryRepository
```

Comment:
Sorry, I agree with them on [Rule 7](https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md#7-do-use-pascalcasing-or-camelcasing-depending-on-the-identifier-type-for-abbreviations-3-characters-or-more-2-chars-are-both-uppercase-when-pascalcasing-is-appropriate-or-inside-the-identifier).
"Consistent with the Microsoft's .NET Framework. Caps would grab visually too much attention."

Remember, it's ok for 2 characters, but not for 3 or more chars.